### DISCUSSION TITLE
Start AND End Anchor for a Floating Bar Chart.

### DISCUSSION THREAD

**Discussion by Aliervo** [chartjs/chartjs-plugin-datalabels#239] [Q&A] [answered]
Unless I am reading the docs wrong, there are only three options for anchoring the data labels: center, start, and end.

I am making a few floating bar charts where I would like the start and end points labeled, but I can't figure out how to position the labels individually; they are always stacked on top of each other.  I am basically just looking for a combination of the start and end anchors, one for each respective point.  

Did I miss something, or does this feature not currently exist?

**Comment by stockiNail** [comment]
@Aliervo do you need something like the following?

![fbAnddatalabels](https://user-images.githubusercontent.com/11741250/115849168-e86a7180-a424-11eb-867e-eaf575cec35c.png)

**Comment by jasonlfunk** [comment]
I don't know about @Aliervo , but this is exactly what I need! :)

**Comment by simonbrunel** [comment, marked as answer]
@Aliervo @jasonlfunk you have to define multiple labels as explained in [the docs](https://chartjs-plugin-datalabels.netlify.app/guide/labels.html#multiple-labels):

```js
plugins: {
  datalabels: {
    labels: {
      start: {
        anchor: 'start',
        formatter: (v) => v[0]
      },
      end: {
        anchor: 'end',
        formatter: (v) => v[1]
      }
    }
  }
}
```

See this [jsfiddle example](https://jsfiddle.net/simonbrunel/831wtez0/).

**Reply by Aliervo** [reply]
That's exactly what I needed, thanks!

I would still like to suggest possibly having this as a shipped option, since the two labels appear by default.  Should I open up a new issue for that?

**Comment by cleejane** [comment]
@simonbrunel Hi, by any chance you can help on this? I managed to display the label with the code you gave but i want to display number as comma separated. how do I add another formatter?
